src/feature.js

```diff
@@ -44,6 +44,10 @@ var feature = function (arg) {
       m_dependentFeatures = [],
       m_selectedFeatures = [];
 
+  // subclasses can add keys to this for styles that apply to subcomponents of
+  // data items, such as individual vertices on lines or polygons.
+  this._subcomponentStyles = {};
```

Comment:
It really is intended to be part of the private API, so it should only matter to developers.  This is the list of styles that are per-vertex rather than per-feature when there is a distinction between vertices and features (i.e., not on point features).  There could be a feature where the sub-feature is not a vertex but something else, so this isn't strictly a list of vertex styles.  I don't know what a better name would be.  `perFeatureStyle` is definitely wrong, as these apply at a finer granularity than features.  `_subfeatureStyles` might be better.  `_stylesAtFinerGranularityThanPerFeature` is too long.